<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>678</height>
   </rect>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>212</x>
         <y>78</y>
         <width>681</width>
         <height>383</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1358</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="actionStop"/>
    <addaction name="actionStep"/>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menu_Visualization">
    <property name="title">
     <string>&amp;Visualization</string>
    </property>
    <addaction name="actionPartition"/>
    <addaction name="actionDensity"/>
    <addaction name="actionVectorField"/>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menu_Visualization"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDummy"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionReload"/>
   <addaction name="actionStep"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
   <addaction name="separator"/>
   <addaction name="actionPartition"/>
   <addaction name="actionDensity"/>
   <addaction name="actionVectorField"/>
   <addaction name="actionPreview"/>
   <addaction name="actionStatistics"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dwStatistics">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Statistics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dwRobotManager">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>44</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Robots</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/document-reload.png</normaloff>:/icons/icons/document-reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/step.png</normaloff>:/icons/icons/step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/export.png</normaloff>:/icons/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export as TeX/TikZ picture</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/zoom-in.png</normaloff>:/icons/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/zoom-out.png</normaloff>:/icons/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/document-new.png</normaloff>:/icons/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/document-save.png</normaloff>:/icons/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="toolTip">
    <string>Save Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>dummy</string>
   </property>
  </action>
  <action name="actionVectorField">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/vector-field.png</normaloff>:/icons/icons/vector-field.png</iconset>
   </property>
   <property name="text">
    <string>Vector Field</string>
   </property>
   <property name="toolTip">
    <string>Show vector field</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/stats.png</normaloff>:/icons/icons/stats.png</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="toolTip">
    <string>Show statistics</string>
   </property>
  </action>
  <action name="actionDensity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/hsv.png</normaloff>:/icons/icons/hsv.png</iconset>
   </property>
   <property name="text">
    <string>Density</string>
   </property>
   <property name="toolTip">
    <string>Show density</string>
   </property>
  </action>
  <action name="actionPartition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/partition.png</normaloff>:/icons/icons/partition.png</iconset>
   </property>
   <property name="text">
    <string>Partition</string>
   </property>
   <property name="toolTip">
    <string>Show Partition</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/record.png</normaloff>:/icons/icons/record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>Create snapshot of each iteration</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/document-save-as.png</normaloff>:/icons/icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save as new document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/icons/eye.png</normaloff>:/icons/icons/eye.png</iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="toolTip">
    <string>Preview trajectory of selected robot</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
